modules/wordads/wordads.php

```diff
@@ -61,6 +61,23 @@ class WordAds {
 		'inline-plugin' => 320,
 	);
 
+	/**
+	 * Counter to enable unique, sequential section IDs for all amp-ad units
+	 *
+	 * @var int
+	 */
+	public static $amp_section_id = 0;
+
+	/**
+	 * Increment the AMP section ID and return the value
+	 *
+	 * @return int
+	 */
+	public static function get_amp_section_id() {
+		self::$amp_section_id++;
+		return self::$amp_section_id;
```

Comment:
return self::++$amp_section_id; would be a little cleaner, but not big deal.

Reply:
Assuming you meant `return self::$amp_section_id++;`, done in https://github.com/Automattic/jetpack/pull/12733/commits/b0fc5def3fd6c884fe8ddb2ed4f876203091716c